From: Sophie Brun <sophie@kali.org>
Date: Tue, 30 Sep 2025 12:27:34 +0200
Subject: Do not build for kernel > 6.15

For Kernel 6.16 or later, there is no need to install this driver and
the build fails: fix the dkms.conf
---
 dkms.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dkms.conf b/dkms.conf
index d4c2f85..e087336 100644
--- a/dkms.conf
+++ b/dkms.conf
@@ -6,4 +6,5 @@ PROCS_NUM=`nproc`
 [ $PROCS_NUM -gt 16 ] && PROCS_NUM=16
 DEST_MODULE_LOCATION[0]="/updates"
 MAKE="'make' -j$PROCS_NUM KVER=${kernelver} KSRC=/lib/modules/${kernelver}/build"
+BUILD_EXCLUSIVE_KERNEL_MAX="6.15"
 AUTOINSTALL="yes"
